angel_system/global_step_prediction/global_step_predictor.py

```diff
@@ -242,69 +344,71 @@ def process_new_confidences(self, activity_confs):
             # activation threshold for enough consecutive frames
             flipping_on_indexes = np.intersect1d(
                 deactivated_indexes,
-                np.where(self.activated_activities[:, 1] >= self.threshold_frame_count)[
-                    0
-                ],
+                np.nonzero(
+                    self.activated_activities[:, 1] >= self.threshold_frame_count
+                )[0],
             )
-            # flipping_on_indexes = np.where((self.activated_activities[:,0] == 0,)
-            #    and (self.activated_activities[:,1] >= self.
-            #        threshold_frame_count,))[1]
             if 1 in flipping_on_indexes:
                 print(f"woo i = {i}")
```

Comment:
Ha, was this debugging or should this be kept around?

Reply:
Woo! Yes. I'll remove that. There's a "boo" somewhere that I'll remove too.